src/stores/SpaceStore.tsx

```diff
@@ -120,7 +120,19 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
      * should not be done when the space switch is done implicitly due to another event like switching room.
      */
     public async setActiveSpace(space: Room | null, contextSwitch = true) {
-        if (space === this.activeSpace || (space && !space?.isSpaceRoom())) return;
+        if (!space?.isSpaceRoom()) return;
```

Comment:
```suggestion
        if (space && !space.isSpaceRoom()) return;
```

Sorry, we ended up deciding this would be better, because otherwise, we would always quit when switching to the home space